app/src/main/java/com/owncloud/android/ui/activity/ToolbarActivity.java

```diff
@@ -158,41 +165,29 @@ private void showHomeSearchToolbar(String title, boolean isRoot) {
 
     @SuppressLint("PrivateResource")
     private void showHomeSearchToolbar(boolean isShow) {
-        viewThemeUtils.material.themeToolbar(mToolbar);
-        if (isShow) {
-            viewThemeUtils.platform.resetStatusBar(this);
-            mAppBar.setStateListAnimator(AnimatorInflater.loadStateListAnimator(mAppBar.getContext(),
-                                                                                R.animator.appbar_elevation_off));
-            mDefaultToolbar.setVisibility(View.GONE);
-            mHomeSearchToolbar.setVisibility(View.VISIBLE);
-            viewThemeUtils.material.themeCardView(mHomeSearchToolbar);
-            viewThemeUtils.material.themeSearchBarText(mSearchText);
-        } else {
-            mAppBar.setStateListAnimator(AnimatorInflater.loadStateListAnimator(mAppBar.getContext(),
-                                                                                R.animator.appbar_elevation_on));
-            viewThemeUtils.platform.themeStatusBar(this);
-            mDefaultToolbar.setVisibility(View.VISIBLE);
-            mHomeSearchToolbar.setVisibility(View.GONE);
-        }
+        mAppBar.setStateListAnimator(AnimatorInflater.loadStateListAnimator(mAppBar.getContext(),
+                                                                            R.animator.appbar_elevation_on));
+        viewThemeUtils.platform.themeStatusBar(this);
+        mDefaultToolbar.setVisibility(View.VISIBLE);
+        mHomeSearchToolbar.setVisibility(View.GONE);
     }
 
     /**
      * Updates title bar and home buttons (state and icon).
      */
     public void updateActionBarTitleAndHomeButtonByString(String title) {
-        String titleToSet = getString(R.string.app_name);    // default
```

Comment:
Code modified as per backport PR: https://github.com/nextmcloud/android/pull/111
Once NC PR (link inside the above PR) is merged we can rebase and this changes will no longer visible here.